lib/tasks/create_fake_data.rake

```diff
@@ -0,0 +1,89 @@
+namespace :db do
+  namespace :seed do
+    desc 'Generate fake patient entries for data wranglers'
+    task :create_fake_data => :environment do
+
+        users = User.all
+        clinics = Clinic.all
+
+        5000.times do |idx|
+          flag = (rand(10) % 5 == 0)
+
+          initial_call = Date.today - rand(1000)
+          
+          has_appt = rand(3) % 2 == 0
+
+          has_pledge = rand(3) != 2 
+
+          lines = ['DC', 'VA', 'MD'] # need to add Spanish maybe? 
+
+          patient = Patient.create!(
+            name: 'Randomized Patient',
+            primary_phone: SecureRandom.random_number(10**10).to_s.rjust(10, '0'), 
```

Comment:
Since we require phone numbers to be unique, there's a slim but real chance that this will cause conflicts and error out. Your use of `rjust` is really inspired though. Can you think of a way that you can produce 5k unique valid phone numbers in a sequence? Something like `999-99X-XXXX` would work. (Let me know if that makes sense, I might notbe explaining it well.)

Reply:
So `'#{idx}'.rjust(10, '9')`, which would produce 999-999-XXXX, creates the same phone number for indeces 9, 99, and 999! -_- I'll keep looking around to see what else can create unique valid phone numbers in a sequence, but for now I kept the SecureRandom.random_number thing, but added the index as the second argument to the `rjust`, which I think should make it more unique?